diff --git a/src/Person.php b/src/Person.php
new file mode 100644
index 0000000..81ee914
--- /dev/null
+++ b/src/Person.php
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace CodeDiff;
+
+
+class Person
+{
+
+    /**
+     * @var string
+     */
+    private $name;
+
+    /**
+     * Person constructor.
+     * @param string $name
+     */
+    public function __construct(string $name)
+    {
+        $this->name = $name;
+    }
+
+    /**
+     * @return string
+     */
+    public function getName(): string
+    {
+        return $this->name;
+    }
+
+}
