diff --git a/src/Printer.php b/src/Foo.php
similarity index 100%
rename from src/Printer.php
index a86c17c..3d4462c 100644
--- a/src/Printer.php
+++ b/src/Foo.php
@@ -6,7 +6,7 @@ declare(strict_types=1);
 namespace CodeDiff;


-class User
+class Person
 {

     /**
@@ -20,7 +20,7 @@ class User
     private $isAdmin;

     /**
-     * User constructor.
+     * Person constructor.
      * @param string $name
      * @param bool $isAdmin
      */
